### PR TITLE
update readme files to have proper formatting in OH docu site

### PR DESCRIPTION
**Pull request by marcelrv**
https://github.com/openhab/openhab2-addons/issues/956

 
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
You're the best - many thanks again!

File: protocols/enocean/README.md [hunk 1]
```diff
@@ -81,7 +81,8 @@ Note
 ----
 Using the example app/bundle (org.eclipse.smarthome.protocols.enocean.sample.client) requires to 
 put the following lines in the enocean_config.txt configuration file of ``org.eclipse.smarthome.
-protocols.enocean.basedriver.impl```:
+protocols.enocean.basedriver.impl```:
```
Comment (maggu2810) [inline protocols/enocean/README.md]: Are there two opening and three closing ones?

Comment (marcelrv) [inline protocols/enocean/README.md]: yes, looks so. Guess a typo in Kai's earlier commit
https://github.com/eclipse/smarthome/blame/ae20c3fa64baa95e50b551a25ee0ee37d78649b7/protocols/enocean/README.md#L83

Comment (marcelrv) [inline protocols/enocean/README.md]: anyway, made fix for it #1566

**Pull request merged**